### DISCUSSION TITLE
Not able to Add Recorder Menu On Ubuntu

### DISCUSSION THREAD

**Discussion by SriShyena** [AlexandreRouma/SDRPlusPlus#911] [General]
I have downloaded 1.0.4 SDRpp and successfully complied the code. I'm unable to add Recorder and other modules to the menu. Can anyone help on this subject?

**Comment by AlexandreRouma** [comment]
old version, get the [nightly](https://www.sdrpp.org/nightly)

Also "unable to add Recorder and other modules to the menu" doesn't say what you're trying to do or what the error is. Show screenshot and describe your problem in a way that people can understand and replicate the actions you are taking.

**Reply by SriShyena** [reply]
Thank you Alexandre, 

We are working on a unique product which will have multi-channel capability. Can we discuss little bit more details?

**Reply by AlexandreRouma** [reply]
Send an email to alexandre.rouma@gmail.com if it's not a public product but do know that:
1) SDR++ only supports one input IQ stream (aka just one waterfall with one SDR) at a time. The multi-vfo features can only add demodulators to that single IQ stream. This means that even if you have multiple records, if they are set to baseband mode, they can only record the one raw IQ stream. Of course with the radio set to raw you can have multiple recorders to record multiple subbands.
2) Since I assume the goal is to use this software with custom hardware, make sure to take notice of the GPLv3 license.
3) I insist on implementing and maintaining support for hardware myself since I take responsability for the stability and performance of the software. This means feature pull requests (eg. a source module) are currently not accepted.
